assets/test/spec/services/applicationGeneratorSpec.js

```diff
@@ -47,14 +47,14 @@ describe("ApplicationGenerator", function(){
         replicas: 1
       },
       imageName: "origin-ruby-sample",
-      imageTag: "latest", 
+      imageTag: "latest",
       imageRepo: {
-    	    "kind": "ImageRepository",
-    	    "apiVersion": "v1beta1",
+    	    "kind": "ImageStream",
+    	    "apiVersion": "v1beta3",
     	    "metadata": {
     	        "name": "origin-ruby-sample",
     	        "namespace": "test",
-    	        "selfLink": "/osapi/v1beta1/imageRepositories/origin-ruby-sample?namespace=test",
+    	        "selfLink": "/osapi/v1beta3/test/imagestreams/origin-ruby-sample",
     	        "uid": "ea1d67fc-c358-11e4-90e6-080027c5bfa9",
     	        "resourceVersion": "150",
     	        "creationTimestamp": "2015-03-05T16:58:58Z"
```

Comment:
check the remaining structure of the fixture and make sure it matches what image streams actually look like... this doesn't look right to me